OpenOversight/app/models/database_cache.py

```diff
@@ -26,19 +28,35 @@ def db_model_cache_key(update_type: str):
     """
 
     def _cache_key(model: Model):
-        return model_key(model, update_type)
+        return get_model_cache_key(model, update_type)
 
     return _cache_key
 
 
+def get_database_cache_entry(model: Model, update_type: str) -> Any:
+    """Get db.Model entry for key in the cache."""
+    key = get_model_cache_key(model, update_type)
+    if key in DB_CACHE.keys():
+        return DB_CACHE.get(key)
+    else:
+        return None
+
+
 def has_database_cache_entry(model: Model, update_type: str) -> bool:
     """db.Model key exists in cache."""
-    key = model_key(model, update_type)
+    key = get_model_cache_key(model, update_type)
     return key in DB_CACHE.keys()
 
 
-def remove_database_cache_entry(model: Model, update_type: str) -> None:
+def put_database_cache_entry(model: Model, update_type: str, data: Any) -> None:
+    """Put data in cache using the constructed key."""
+    key = get_model_cache_key(model, update_type)
+    DB_CACHE[key] = data
+
+
+def remove_database_cache_entry(model: Model, update_type: List[str]) -> None:
```

Comment:
Should it be `update_types` (plural) since it seems to expect a list of strings?